common/djangoapps/nodebb/apps.py

```diff
@@ -8,4 +11,4 @@ def ready(self):
         """
         Connect signal handlers.
         """
-        import nodebb.signals.handlers
+        import nodebb.signals.handlers  # pylint: disable=unused-variable
```

Comment:
@annas-usman I would suggest fixing the issue rather than disabling the linting for lines. If you want to discuss the solution feel free to contact.

Reply:
As discussed in our meeting and discussion with Ahmed Faraz, I have decided to keep this suppression, because if we we are to remove this suppression, a lot of changes would have to be made, including modifying apps.py, models.py and handlers.py, by removing receiver statements, importing all models using apps.get_model and adding signal.connect() statement for all 17 handlers, hence it would require extensive testing.